platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -3512,6 +3533,11 @@
             $('#importInstanceModal').modal('show');
         };
 
+        $scope.deleteSelectedInstances = function() {
+            $scope.checkedEntitiesJSON = $scope.getCheckedEntitiesJSON();
+            $("#deleteSelectedInstancesModal").modal('show');
```

Comment:
Ideally this modal would use BootstrapDialog3 and would be tied into the modal window sequencing // but this will work for now, and we will change this when the MDS module is moved to MOTECH-UI